### ISSUE TITLE
RddLeto, add table/index folder structure error.

### ISSUE THREAD

**Issue by hansmarc** [hansmarc/Otis#2]
There are some tree errors with multiple folders and multiple subfolders in the Leto data folder. When you add a table or a index file, my function leto_getfile() presents sometimes a incorrect folder / subfolder tree in the left part of this screen.

**Comment by sapomuyverde** [comment]
Hi,
Could I contact you? my mail is jparada_a[at]hotmail[dot]com

Regards,
Javier

**Comment by hansmarc** [comment]
Hello Javier,

Sure you can contact me, no problem,
How can i help you ?

Regards
Hans

Envoyé

**Comment by sapomuyverde** [comment]
Hi Hans,

I appreciate that you get in touch.

My interest in contacting you is because I understand that your software
called Otis has been developed with the hwgui GUI, is that correct ?, and
personally I am very interested in testing that GUI, since at the moment I
know and have programmed small modules with hmg and oohg, but my interest
has always been to try hwgui, only that the times that I have tried it I
have not even been able to compile the project and generate the libraries
and when I have downloaded the binaries the interface of the examples is
displayed with an old style like if it was Windows 95 (some issue with the
Windows.Manifest file, I think).

And looking at Otis I can see that you have a good product with an
interface that I personally like, so I would like to try hwgui (again), and
above all my interest is to test the browse/grid to display sales analysis
data that I get from the sql server that in my case the database engine is
ms-sql server.

So basically I want to contact you and see if you have enough free time to
try to help me compile hwgui and the examples with the correct look (not
old)

I don't know if with hwgui I can make use of Activex, (the browser) since
the project I have in mind includes graphics and those graphics are dynamic
so I need to put the Activex control in a form to display that part, I
already have all that solved, it would just be that hwgui also implements
Activex.

And this is the grid or browse control?

[image: img1.png]

That you have in Otis that I am also very interested in testing, I like the
interface and I see that it can even be totalized, is that correct?

So Hans excuse me for the length of my mail, but I want to put you in
context of what I would like to test/develop with hgwui.

I appreciate your time and your help.

Regards,
Javier

**Comment by hansmarc** [comment]
Hi Javier,

 

Sorry for the late response.

 

I think that you made a mistake. I didn’t write Otis with hwgui but i used

minihmg extended, see the next link http://www.hmgextended.com/.

I suppose you know the library and site.

 

It is a fork off the official hmg, see link https://www.hmgforum.com/.

Some years ago i compared some libraries and i use one of the two libs above

in function of projects.

 

I have no experience with hwgui nor with ActiveX calls. 

If i am not mistaken hwgui is written by Alexander Kresin.

 

So i am afraid i can’t help you because of the mentionned reasons above.

 

Greatings

Hans

 

 

De : Javier Parada <notifications@github.com> 
Envoyé : vendredi 8 janvier 2021 18:54
À : hansmarc/Otis <Otis@noreply.github.com>
Cc : Hans <hansmarc@skynet.be>; Author <author@noreply.github.com>
Objet : Re: [hansmarc/Otis] RddLeto, add table/index folder structure error. (#2)

 

Hi Hans,

I appreciate that you get in touch.

My interest in contacting you is because I understand that your software
called Otis has been developed with the hwgui GUI, is that correct ?, and
personally I am very interested in testing that GUI, since at the moment I
know and have programmed small modules with hmg and oohg, but my interest
has always been to try hwgui, only that the times that I have tried it I
have not even been able to compile the project and generate the libraries
and when I have downloaded the binaries the interface of the examples is
displayed with an old style like if it was Windows 95 (some issue with the
Windows.Manifest file, I think).

And looking at Otis I can see that you have a good product with an
interface that I personally like, so I would like to try hwgui (again), and
above all my interest is to test the browse/grid to display sales analysis
data that I get from the sql server that in my case the database engine is
ms-sql server.

So basically I want to contact you and see if you have enough free time to
try to help me compile hwgui and the examples with the correct look (not
old)

I don't know if with hwgui I can make use of Activex, (the browser) since
the project I have in mind includes graphics and those graphics are dynamic
so I need to put the Activex control in a form to display that part, I
already have all that solved, it would just be that hwgui also implements
Activex.

And this is the grid or browse control?

[image: img1.png]

That you have in Otis that I am also very interested in testing, I like the
interface and I see that it can even be totalized, is that correct?

So Hans excuse me for the length of my mail, but I want to put you in
context of what I would like to test/develop with hgwui.

I appreciate your time and your help.

Regards,
Javier
